src/util/unicode.cpp

```diff
@@ -256,8 +256,14 @@ const char **narrow_argv(int argc, const wchar_t **argv_wide)
   return argv_narrow;
 }
 
-std::wstring utf8_to_utf16(const std::string& in)
+std::wstring utf8_to_utf16be(const std::string& in)
 {
   std::wstring_convert<std::codecvt_utf8_utf16<wchar_t> > converter;
   return converter.from_bytes(in);
 }
+
+std::wstring utf8_to_utf16le(const std::string& in)
+{
+  std::wstring_convert<std::codecvt_utf8_utf16<wchar_t,0x10ffffUL,std::codecvt_mode::little_endian> > converter;
```

Comment:
Unfortunately I couldn't find such a constant, this is in fact the default for this template, hopefully this commit https://github.com/diffblue/cbmc/pull/372/commits/af792fe4d7e7f658317c1167457f31bc9b88278e makes it a bit clearer